package.json

```diff
@@ -2,7 +2,7 @@
     "license": "MIT",
     "repository": "https://github.com/sulu/sulu-minimal.git",
     "scripts": {
-        "build": "webpack -p",
+        "build": "rimraf web/adminV2 && webpack -p",
```

Comment:
should we introduce an additional folder here removing all under /admin sounds for me not a good idea if someone places here something. /cc @sulu/core-team